curp/src/client.rs

```diff
@@ -19,6 +19,8 @@ use crate::{
 
 /// Protocol client
 pub struct Client<C: Command> {
+    /// client id
+    id: ServerId,
```

Comment:
In the comment, should it be local server id?

Reply:
How come?